antlr/PlantUML.g4

```diff
@@ -257,6 +261,8 @@ identifier
     | 'end'
     | 'as'
     | 'scale'
+    | 'skin'
+    | 'mainframe'
```

Comment:
I recently forgot about this on another plantuml PR. Nice catch! Really appreciate it :)